README.md

```diff
@@ -6,69 +6,25 @@ This repository is the home of Pulp's CI files.
-
-### Pulp Gem
-
-* [![Pulp Nightly CI/CD](https://github.com/pulp/pulp_gem/actions/workflows/nightly.yml/badge.svg)](https://github.com/pulp/pulp_gem/actions/workflows/nightly.yml)
-* [![Pulp Release CI/CD](https://github.com/pulp/pulp_gem/actions/workflows/release.yml/badge.svg)](https://github.com/pulp/pulp_gem/actions/workflows/release.yml)
-
-### Pulp Maven
-
-* [![Pulp Nightly CI/CD](https://github.com/pulp/pulp_maven/actions/workflows/nightly.yml/badge.svg)](https://github.com/pulp/pulp_maven/actions/workflows/nightly.yml)
-* [![Pulp Release CI/CD](https://github.com/pulp/pulp_maven/actions/workflows/release.yml/badge.svg)](https://github.com/pulp/pulp_maven/actions/workflows/release.yml)
-
-### Pulp NPM
-
-* [![Pulp Nightly CI/CD](https://github.com/pulp/pulp_npm/actions/workflows/nightly.yml/badge.svg)](https://github.com/pulp/pulp_npm/actions/workflows/nightly.yml)
-* [![Pulp Release CI/CD](https://github.com/pulp/pulp_npm/actions/workflows/release.yml/badge.svg)](https://github.com/pulp/pulp_npm/actions/workflows/release.yml)
-
-### Pulp RPM
-
-* [![Pulp Nightly CI/CD](https://github.com/pulp/pulp_rpm/actions/workflows/nightly.yml/badge.svg)](https://github.com/pulp/pulp_rpm/actions/workflows/nightly.yml)
-* [![Pulp Release CI/CD](https://github.com/pulp/pulp_rpm/actions/workflows/release.yml/badge.svg)](https://github.com/pulp/pulp_rpm/actions/workflows/release.yml)
-
-### Certguard
-
-* [![Pulp Nightly CI/CD](https://github.com/pulp/pulp-certguard/actions/workflows/nightly.yml/badge.svg)](https://github.com/pulp/pulp-certguard/actions/workflows/nightly.yml)
-* [![Pulp Release CI/CD](https://github.com/pulp/pulp-certguard/actions/workflows/release.yml/badge.svg)](https://github.com/pulp/pulp-certguard/actions/workflows/release.yml)
-
-### Pulp CLI
-
-* [![Nightly Test](https://github.com/pulp/pulp-cli/actions/workflows/nightly.yml/badge.svg)](https://github.com/pulp/pulp-cli/actions/workflows/nightly.yml)
-* [![Pulp CLI Release](https://github.com/pulp/pulp-cli/actions/workflows/release.yml/badge.svg)](https://github.com/pulp/pulp-cli/actions/workflows/release.yml)
-
-### Pulp Python
-
-* [![Pulp Nightly CI/CD](https://github.com/pulp/pulp_python/actions/workflows/nightly.yml/badge.svg)](https://github.com/pulp/pulp_python/actions/workflows/nightly.yml)
-* [![Pulp Release CI/CD](https://github.com/pulp/pulp_python/actions/workflows/release.yml/badge.svg)](https://github.com/pulp/pulp_python/actions/workflows/release.yml)
-
-### Pulp OSTree
-* [![Pulp Nightly CI/CD](https://github.com/pulp/pulp_ostree/actions/workflows/nightly.yml/badge.svg)](https://github.com/pulp/pulp_ostree/actions/workflows/nightly.yml)
-* [![Pulp Release CI/CD](https://github.com/pulp/pulp_ostree/actions/workflows/release.yml/badge.svg)](https://github.com/pulp/pulp_ostree/actions/workflows/release.yml)
+Plugin | Nightly | CI Update
+:---|---:|---:
```

Comment:
Could we center the Nightly/CI-Update headers? Maybe it's just me, feels a little odd right-justifying them?

Reply:
I don't know that i have that kind of control with markdown tables. But we can left justify / center everything?